docs/drupal/multisite.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
See https://github.com/ProboCI/probo.ci/pull/147 for historical info about these docs. I am still unsure they are 100% accurate.

Reply:
Just tested, and it appears this works with a few modifications.